### PR TITLE
fixed gpu error

### PR DESCRIPTION
**Pull request by adakoda**
#52 modification still cause following error.
->
`Error processing request: can't convert cuda:0 device type tensor to numpy. Use Tensor.cpu() to copy the tensor to host memory first.`

So we need to convert output type to cpu from gpu.

Note:
If you use docker environment, please add -`-gpus all` option.

```
docker run -it --rm --gpus all -p 10000:10000 -p 5005:5005 unity-robotics:pose-estimation /bin/bash
```
You can also use `nvidia-smi` command in docker whether gpu is enable or not.

### CODE REVIEW

**Comment by JonathanLeban** [comment]
thank you @adakoda on your report and your suggestion. What is the line causing the error about the conversion to cuda:0 ?

**Comment by adakoda** [comment]
@JonathanLeban 

> What is the line causing the error about the conversion to cuda:0 ?

`output_translation, output_orientation = output_translation.detach().numpy(), output_orientation.detach().numpy()`

The model return gpu device type and you will get following error message.

`Error processing request: can't convert cuda:0 device type tensor to numpy. Use Tensor.cpu() to copy the tensor to host memory first.`

**Comment by at669** [comment]
Hi @adakoda, this repo is not currently accepting external contributions, as per the [Contributing Guidelines](https://github.com/Unity-Technologies/Robotics-Object-Pose-Estimation/blob/main/CONTRIBUTING.md). For that reason, I will close this PR, but please feel free to continue discussing in this thread, @JonathanLeban and @adakoda. Once resolved, the potential fix will be added via an internal PR. Sorry for the inconvenience!

**Comment by adakoda** [comment]
@at669 @JonathanLeban
Thank you for your works! 

This is the description of the sequence of code.
This notebook is available on Google CoLab for anyone to try.
https://github.com/adakoda/Robotics-Object-Pose-Estimation/blob/fixed_cpu_error/Model/Pose_Estimation_not_working_correctly_45.ipynb

I hope this explanation will be useful to someone.

**Comment by JonathanLeban** [comment]
Hey ! So the problem was solved in the `dev` branch and we are going to merge the `dev` branch with the `main` branch soon. Thank you !!
